src/image.rs

```diff
@@ -145,15 +145,16 @@ impl Data {
             DynamicImage::ImageLumaA8(_) => Format::R8G8,
             DynamicImage::ImageRgb8(_) => Format::R8G8B8,
             DynamicImage::ImageRgba8(_) => Format::R8G8B8A8,
-            DynamicImage::ImageBgr8(_) => Format::B8G8R8,
-            DynamicImage::ImageBgra8(_) => Format::B8G8R8A8,
```

Comment:
Why have these cases been removed?

Reply:
They have been removed from the `DynamicImage` of the `image` crate. You can take a look at the corresponding discussions https://github.com/image-rs/image/pull/1482 https://github.com/image-rs/image/issues/1671